README.md

```diff
@@ -21,3 +21,8 @@ Here's the decisions we taken so far:
 | ADR | Title                         | PR (discussion) |
 | --- | ----------------------------- | --------------- |
 | 1   | [Record architecture decisions](doc/architecture/decisions/0001-record-architecture-decisions.md) | [PR#25](https://github.com/teamdigitale/italia-pagopa-proxy/pull/25)                |
+| 2   | [Use uuid/v1 for sequential requests](doc/adr/0002-use-uuid-v1-for-sequential-requests.md) | [PR#8](https://github.com/teamdigitale/italia-pagopa-proxy/pull/8)                |
+
+## NOTES
+
+This project includes manually @types/bluebird because it's necessary for soap lib. It will be removed when this dependency will be fixed into soap.
```

Comment:
link to the relevant issue (I guess there's an open issue on the `node-soap` repo?)